terraform/files/containerssh_config.yaml

```diff
@@ -0,0 +1,71 @@
+log:
+  level: warning
+ssh:
+  banner: |
+
+    ********************************************************************
+                               Warning!
+    ********************************************************************
+
+    This is a honeypot. All information, including IP address, username,
+    password, any commands you type, or files you upload will be visible
+    to the honeypot.
+
+    If you do not agree disconnect now.
+
+    ********************************************************************
+
+  hostkeys:
+    - /etc/containerssh/ssh_host_rsa_key
+backend: docker
+docker:
+  connection:
+    host: tcp://sacrificial-vm:2376
+    cert: |
+      -----BEGIN CERTIFICATE-----
+      <client certificate here>
+      -----END CERTIFICATE-----
+    key: |
+      -----BEGIN RSA PRIVATE KEY-----
+      <client key here>
+      -----END RSA PRIVATE KEY-----
+    cacert: |
+      -----BEGIN CERTIFICATE-----
+      <CA certificate here>
+      -----END CERTIFICATE-----
+  execution:
+    imagePullPolicy: Never
+    container:
+      image: containerssh-guest-image
+      hostname: bitcoin
+      # Disable network in the container
+      networkdisabled: true
+      # Force running as user 1000
+      user: 1000
+      # Optionally set working directory
+      workingdir: /home/ubuntu
+    host:
+      # Don't let the attacker write to the root FS.
+      readonlyrootfs: true
+      resources:
+        # 10% of CPU
+        cpuperiod: 10000
+        cpuquota: 1000
+        # 50 MB of memory with swap
+        memoryswap: 52428800
+        memoryswappiness: 50
+        # 25 MB of memory
+        memory: 26214400
+        # Reserve 20 MB of memory
+        memoryreservation: 20000000
+        # Max 1000 processes to prevent fork bombs
+        pidslimit: 1000
+      tmpfs:
+        # Create writable directories in memory
+        /tmp: rw,noexec,nosuid,size=65536k,uid=1000,gid=1000
+        /run: rw,noexec,nosuid,size=65536k,uid=1000,gid=1000
+        /home/ubuntu: rw,noexec,nosuid,size=65536k,uid=1000,gid=1000
+auth:
```

Comment:
note: not including `metrics` and `audit` fields for now.
Add them later when they are correctly configured.